dist/index.js

```diff
@@ -693,7 +693,18 @@ async function run() {
     let imageTag = core.getInput('imageTag', { required: false });
     const ref = process.env['GITHUB_REF'];
     const refArray = ref.split('/');
-    if (!imageTag) imageTag = refArray[refArray.length - 1];
+    if (!imageTag) {
+        const refLast = refArray[refArray.length - 1];
+        if (refLast === "merge" && refArray.length >= 2) {
+            imageTag = "mr" + refArray[refArray.length - 2];
+        } else {
+            imageTag = refLast;
+        }
+    }
+    let imageTagPrefix = core.getInput('imageTagPrefix', { required: false });
+    if (imageTagPrefix) imageName = imageTagPrefix + imageName;
```

Comment:
I also made a mistake here (and 2 limes below) 
Should be imageTag not imageName

Reply:
I've addressed this in #12 

Take a look when you get the chance, and I'll put the fix in a new release.